test/test_tensorboard.py

```diff
@@ -462,16 +462,12 @@ def test_torchvision_smoke(self):
                 'vgg19': (2, 3, 224, 224),
                 'vgg16_bn': (2, 3, 224, 224),
                 'vgg19_bn': (2, 3, 224, 224),
-                'mobilenet_v2': (2, 3, 224, 224),  # will fail optimize_graph
+                'mobilenet_v2': (2, 3, 224, 224),
             }
             for model_name, input_shape in model_input_shapes.items():
                 with SummaryWriter(comment=model_name) as w:
                     model = getattr(torchvision.models, model_name)()
-                    # ValueError: only one element tensors can be converted to Python scalars
-                    if model_name == 'mobilenet_v2':
-                        w.add_graph(model, torch.zeros(input_shape), operator_export_type="RAW")
-                    else:
-                        w.add_graph(model, torch.zeros(input_shape))
+                    w.add_graph(model, torch.zeros(input_shape))
```

Comment:
Great!

Reply:
The new trace function without ONNX processes mobilenet without problems.